guides/user/pipeline/managing-pipelines/index.md

```diff
@@ -0,0 +1,146 @@
+triggers.
+<!-- TODO(nhayes): link here to pipeline triggers overview once it exists. -->
+
+## Disable a pipeline
+
+Disabling a pipeline prevents any triggers from firing, as well as preventing
+users from running it manually.
+
+1. From the **Pipelines** tab, click **Configure** to modify an existing
+pipeline.
+  ![](images/select-configure.png)
+2. Click **Pipeline actions** in the upper right corner, and select **Disable**.
+  ![](images/pipeline-actions.png)
+
+In order to re-enable your disabled pipeline, select **Pipeline actions** and
+choose **Enable**.
+
+## Delete a pipeline
+
+1. From the **Pipelines** tab, click **Configure** to modify an existing
+pipeline.
+  ![](images/select-configure.png)
+2. Click **Pipeline actions** in the upper right corner, and select **Delete**.
+  ![](images/pipeline-actions.png)
+
+## Edit a pipeline as JSON
+
+> **Warning**: the JSON you write here is not validated -- Edit as JSON
+essentially allows you to modify the pipeline via a free-form textbox. **Be
+careful!** It is very easy to break the pipeline.
+
+Spinnaker represents pipelines as JSON behind the scenes. Any changes you make
+to your pipeline using the UI are converted to JSON when Spinnaker saves the
+pipeline.
+
+When you use the **Edit as JSON** feature, you are directly editing the payload.
+This can be useful for working around limitations of the UI.
+
+In order to edit your pipeline as JSON:
+
+1. From the **Pipelines** tab, click **Configure** to modify an existing
+pipeline.
+  ![](images/select-configure.png)
+2. Click **Pipeline actions** in the upper right corner, and select
+**Edit as JSON**.
+  ![](images/pipeline-actions.png)
+
+## View and restore revision history
+
+Each time you save your pipeline, the current version is added to revision
+history. You can use revision history to diff two versions of a pipeline or to
+restore an older version of a pipeline.
+
+> *Note*: If you are using minio or redis to store your Spinnaker configuration
+files, you won't be able to use revision history because neither minio nor redis
+supports it.
+
+1. From the **Pipelines** tab, click **Configure** to modify an existing
+pipeline.
+  ![](images/select-configure.png)
+2. Click **Pipeline actions** in the upper right corner, and select
+**View revision history**.
+  ![](images/pipeline-actions.png)
+
+This pulls up a window that shows a JSON representation of your current
+pipeline. From there, you can see all previously saved versions via the
+drop-down menu in the upper left corner.
+
+### Restore a previous version
+
+1. Choose a version from the **Revision** drop-down.
+  ![](images/revision-history.png)
+2. A button appears that allows you to restore your pipeline to that version. If
+you restore an older version, the current version of your pipeline is saved in
+your revision history in case you want to return to it in the future.
+  ![](images/restore-revision.png)
+
+### Diff two versions of a pipeline
```

Comment:
This seems like an option you'd try _before_ restoring to a previous version?

Reply:
Yep, good point (here and above about stages/triggers). I'll open another PR to reorder this and address some of your other comments.